nginx-default.conf.template

```diff
@@ -1,3 +1,4 @@
+map_hash_bucket_size 128;
```

Comment:
What problem does this additional directive solve?

Reply:
nginx refused to start without this. I tried the lower values and it still refused. I included the error in the comment details a489861. I presume it has to do with connection pooling.